packages/block-editor/src/components/link-control/index.js

```diff
@@ -261,23 +266,26 @@ function LinkControl( {
 				/>
 			) }
 
-			<div className="block-editor-link-control__tools">
-				<LinkControlSettingsDrawer
-					value={ value }
-					settings={ settings }
-					onChange={ onChange }
-				/>
-				{ onRemove && value && ! isEditingLink && ! isCreatingPage && (
-					<Button
-						className="block-editor-link-control__unlink"
-						isDestructive
-						variant="link"
-						onClick={ onRemove }
-					>
-						{ __( 'Unlink' ) }
-					</Button>
-				) }
-			</div>
+			{ showSettingsDrawer ||
```

Comment:
Whoops! This introduces a regression. Thank you unit tests (which I wrote 😉 )!

So...let me think about this.

Reply:
Fixed.